.github/workflows/docker-build-static.yml

```diff
@@ -181,34 +176,35 @@ jobs:
           key: lldap-bin-aarch64-${{ hashFiles('**/Cargo.lock') }}
           restore-keys: |
             lldap-bin-aarch64-
-      - name: compile aarch64
-        run: cargo build --target=aarch64-unknown-linux-gnu --release -p lldap -p migration-tool
+#      - name: build
+#        run: ./build-arm64.sh
+      - name: add musl aarch64 target
+        run: rustup target add aarch64-unknown-linux-musl
+      - name: build lldap aarch4
+        run: cargo build --target=aarch64-unknown-linux-musl --release -p lldap -p migration-tool
       - name: check path
-        run: ls -al target/aarch64-unknown-linux-gnu/release/
+        run: ls -al target/aarch64-unknown-linux-musl/release/
       - name: upload aarch64 lldap artifacts
         uses: actions/upload-artifact@v3
         with:
           name: aarch64-lldap-bin
-          path: target/aarch64-unknown-linux-gnu/release/lldap
+          path: target/aarch64-unknown-linux-musl/release/lldap
       - name: upload aarch64 migration-tool artifacts
         uses: actions/upload-artifact@v3
         with:
           name: aarch64-migration-tool-bin
-          path: target/aarch64-unknown-linux-gnu/release/migration-tool
+          path: target/aarch64-unknown-linux-musl/release/migration-tool
 
   build-amd64:
     runs-on: ubuntu-latest
     container:
-      image: rust:1.61
+      image: martadinata666/debian:rust-dev
```

Comment:
What's this image?

Reply:
It my personal rust dev image, with musl gcc included, should be using script, but somehow gtihub actions fail connect to `musl.cc` the steps stuck hour or so. Yet i can fetch it fine. 🙄

Reply:
Yeah, I'd like to avoid external dependencies like that if possible. Could you check it in?

Reply:
the github actions itself used to be build correctly https://github.com/nitnelave/lldap/runs/7481312391?check_suite_focus=true , 9days ago, yet when yesterday i rebase suddenly everything just fail due connection timeout. 🤦🏼 I added `Dockerfile.dev` in case we forced to build one ourself.

Reply:
So i test PR to my self it fetch musl.cc fine https://github.com/martadinata666/lldap/runs/7607098512?check_suite_focus=true 🤦🏼 i just dont know what to say.

Reply:
Maybe try to download it from a github-hosted mirror? e.g. https://github.com/bminor/musl/releases/tag/v1.2.3

Reply:
If needed, we can check it in.

Reply:
So i think we gonna use the custom image, i pushed code for aarch64 to fetch musl-gcc from `musl.cc` and both actions, your repo and mine stuck at it. Im gonna comment manual fetch, and use custom image until this actions works correctly.

Reply:
If we do go the way of the custom image, then let's check in the dockerfile, and I'll build and release it under `nitnelave/` to consolidate the project dependencies.

Reply:
I already add the dockerfile named `Dockerfile.dev`, but maybe easier copied here.
```
FROM rust:1.62-slim-bullseye
# Set needed env path
ENV PATH="/opt/aarch64-linux-musl-cross/:/opt/aarch64-linux-musl-cross/bin/:/opt/x86_64-linux-musl-cross/:/opt/x86_64-linux-musl-cross/bin/:$PATH"

### Install build deps x86_64
RUN apt update && \
    apt install -y --no-install-recommends curl git wget build-essential make perl pkg-config curl tar jq musl-tools && \
    curl -fsSL https://deb.nodesource.com/setup_lts.x | bash - && \
    apt update && \
    apt install -y --no-install-recommends nodejs && \
    apt clean && \
    rm -rf /var/lib/apt/lists/* && \
    npm install -g npm && \
    npm install -g yarn && \
    npm install -g pnpm

### Install build deps aarch64 build
RUN dpkg --add-architecture arm64 && \
    apt update && \
    apt install -y gcc-aarch64-linux-gnu g++-aarch64-linux-gnu libc6-arm64-cross libc6-dev-arm64-cross && \
    apt clean && \
    rm -rf /var/lib/apt/lists/* && \
    rustup target add aarch64-unknown-linux-gnu

### Install build deps armhf build
#RUN dpkg --add-architecture armhf && \
#    apt update && \
#    apt install -y gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf libc6-armhf-cross libc6-dev-armhf-cross && \
#    apt clean && \
#    rm -rf /var/lib/apt/lists/*
### add armhf target
#RUN rustup target add rustup target add armv7-unknown-linux-gnueabihf

### Add musl-gcc aarch64 and x86_64
RUN wget -c https://musl.cc/x86_64-linux-musl-cross.tgz && \
    tar zxf ./x86_64-linux-musl-cross.tgz -C /opt && \
    wget -c https://musl.cc/aarch64-linux-musl-cross.tgz && \
    tar zxf ./aarch64-linux-musl-cross.tgz -C /opt && \
    rm ./x86_64-linux-musl-cross.tgz && \
    rm ./aarch64-linux-musl-cross.tgz

CMD ["bash"]
```

Reply:
Alright, I just pushed `nitnelave/rust-dev:latest` based on this dockerfile. Feel free to use that.